### PR TITLE
Module2

### PR DESCRIPTION
**Pull request by JonnyHarris94**
Please make sure you are raising this PR against your own repository and not the original. If it says https://github.com/CorndelWithSoftwire in the search bar right now, you are in the wrong place!

Please also don't forget to submit the PR's url to Aptem afterwards (by clicking on the corresponding exercise component, pasting the url in the textbox that appears and then pressing the finish button).

### CODE REVIEW

File: todo_app/app.py
Comment (BenRamchandani) [inline todo_app/app.py]: This line of imports isn't used. In fact you can get rid of `session_items.py` completely

File: todo_app/data/trello_items.py
Comment (BenRamchandani) [inline todo_app/data/trello_items.py:12]: You should try and avoid hard coding things like the board id (N46znsdT) so that other people can develop your app. You could make this an environment variable like `TrelloApi`.

This is not required for module 2, but you'll need to do it for module 3.

**Review by BenRamchandani — CHANGES_REQUESTED** [review]
Hi, what you've done looks good, but I think you've missed Part 5 of the exercise "Create a class for to-do items".

Can you have a go at that? Let us know if you need a hand. Can you also update your `README.md` with setup instructions as well as `.env.template` with placeholders for the environment variables you've introduced.

Before you resubmit please also have a look at Hugh's comments on https://github.com/JonnyHarris94/DevOps-Academy/pull/1 and merge that in.

File: README.md
Comment (JackMeadDev) [inline README.md:35]: If this was a real project I'd definitely recommend a bit more detail on how someone could get an appropriate token/Api key

**Review by JackMeadDev — APPROVED** [review]
Thanks for making those changes, I'm happy for you to merge this when you're ready now that class is being used. I can't see any changes to the `.env.template` file that I saw you make, so that might not have been checked in - it would still be worthwhile adding that change before merging, but no need for a re-review, or feel free to include it with your M3 submission.

**Pull request merged**
